compiler/noirc_frontend/src/elaborator/comptime.rs

```diff
@@ -40,6 +40,8 @@ struct AttributeContext {
     attribute_module: LocalModuleId,
 }
 
+type CollectedAttributes = Vec<(FuncId, Value, Vec<Expression>, AttributeContext, Span)>;
```

Comment:
Totally optional: should this tuple be its own struct? This is the only comment I have on this PR, and I'm not even sure a struct is needed here (it's kind of clear what each of these is because of the types) but it might slightly improve readability.

Reply:
I think it's fine as an alias mostly because it's just something private to this module to reduce typing / clarify signatures a bit. I don't think anything would be gained from making it a struct in that case.